src/abhishekSJun20/CharacterOccuranceInString.java

```diff
@@ -0,0 +1,26 @@
+package abhishekSJun20;
+
+public class CharacterOccuranceInString {
+
+	static void countChars(String input) {
+		for (int index = 0;index<input.length();index++) {
+			int count = 0;
+			if (index == input.indexOf(input.charAt(index))) {
+				for (int innerIndex = 0; innerIndex < input.length(); innerIndex++) {
+					if (input.charAt(index) == input.charAt(innerIndex))
```

Comment:
You have increased the complexity in code, Using one for loop you can achieve the output.